release/pipeline/airflow_scripts.sh

```diff
@@ -23,7 +23,7 @@ SCRIPTPATH=$( pwd -P )
 # shellcheck source=release/pipeline/gcb_build_lib.sh
 source "${SCRIPTPATH}/gcb_build_lib.sh"
 
-# Helper function not called by Airflow, used only by functions in this file
+# Helper function called by Airflow before calling the actual work functions below
 function create_subs_file() {
```

Comment:
if this function is always called, you may as well remove the function and put the few lines in top level under the above source "..."

Reply:
done